### PR TITLE
captures the most common parameters and identities in "schedule-status" grouping

### PR DESCRIPTION
**Pull request by QiufangMa**

### CODE REVIEW

File: draft-ma-opsawg-schedule-yang.md [hunk 1]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
    The "schedule-id" parameter is useful to uniquely identify a schedule in
    a network device or controller if multiple scheduling contexts exists.
 
-   The "state" parameter is defined to configure/expose the sheduling state, depends
-   on the use of the grouping. Likewise, the "version" parameter summarizes the
-   current schedule version information.
+   The "state" parameter is defined to configure/expose the sheduling state,
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:235]:
```suggestion
   The "state" parameter is defined to configure/expose the scheduling state,
```

File: draft-ma-opsawg-schedule-yang.md [hunk 2]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   depending on the use of the grouping. The "identityref" type is used for this
+   parameter to allow extensibility in future modules. For example, a "conflicted"
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:237]:
```suggestion
   parameter to allow extensibility in future modules. For example, a "conflict"
```

File: draft-ma-opsawg-schedule-yang.md [hunk 3]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   state is valid in scheduling contexts where multiple systems struggle for the
+   scheduling of the same property.
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:239]:
```suggestion
   scheduling of the same property. The conflict may be induced by, e.g., multiple entities managing the schedules for the same target component.
```

File: draft-ma-opsawg-schedule-yang.md [hunk 4]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
 
-   The "last-update" parameter identifies when the schedule was last modified.
+   The "version" parameter summarizes the current schedule version information,
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:241]:
```suggestion
   The "version" parameter is used to track the current schedule version information. The version can be bumped by the entity who create the schedule.
```

File: draft-ma-opsawg-schedule-yang.md [hunk 5]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   and the "last-update" parameter identifies when the schedule was last modified.
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:242]:
```suggestion
   The "last-update" parameter identifies when the schedule was last modified. In some contexts, this parameter can be
   used to track the configuration of a given schedule. In such cases, the "version" is not used.
```

File: draft-ma-opsawg-schedule-yang.md [hunk 6]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
 
    The "schedule-type" parameter identifies the type of the current schedule.
    The "counter", "last-occurrence", and "upcoming-occurrence" data nodes are
    only avaliable when the "schedule-type" is "recurrence".
 
+   The current grouping captures the most common definition that is applicable
+   to all scheduling contexts. Future modules can define other useful
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:249]:
```suggestion
   to typical scheduling contexts known so far. Future modules can define other useful
```

File: draft-ma-opsawg-schedule-yang.md [hunk 7]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   parameters as needed. For example, in scheduling context that allows multiple
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:250]:
```suggestion
   parameters as needed. For example, in a  scheduling context with multiple
```

File: draft-ma-opsawg-schedule-yang.md [hunk 8]
```diff
@@ -232,16 +232,24 @@ Also, this document uses the YANG terminology defined in {{Section 3 of !RFC7950
...
+   system sources to exist, the "source" and "precedence" parameters may need to
```
Comment (boucadair) [inline draft-ma-opsawg-schedule-yang.md:251]:
```suggestion
   system sources to feed the schedules, the "source" and "precedence" parameters may need to
```

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-ma-opsawg-schedule-yang.md (hunk 5 above)
Comment (QiufangMa) [reply draft-ma-opsawg-schedule-yang.md:242]: > In such cases, the "version" is not used.

In such cases, the "version" may not be used. 
Better? i think these two parameters are  not conflicting. An iteration version number helps obtain the number of historical modifications, while last-update can be used to understand when it was last modified.

Comment (boucadair) [reply draft-ma-opsawg-schedule-yang.md]: Ok for "may not be" for now. Edited the change  accordingly. 

I expect we will have this comment during the publication process to clarify the relationship between the two.

**Comment by boucadair** [comment]
This fixes some of the issues in #79

File: draft-ma-opsawg-schedule-yang.md (hunk 5 above)
Comment (QiufangMa) [reply draft-ma-opsawg-schedule-yang.md]: Okay, sure, I am okay to drop "version" parameter if there is any objection in the future.

**Pull request merged**